docs/developer-docs/latest/development/plugins/i18n.md

```diff
@@ -0,0 +1,575 @@
+---
+title: Internationalization (i18n) - Strapi Developer Documentation
+description: Instructions on how to use Strapi Content API with the Internationalization (i18n) optional plugin
+---
+
+# Internationalization (i18n)
+<!-- ? wondering if it would be a good idea to add the 🌍 icon to the h1 — if yes, we should do this for the h1 of other plugins as well -->
+
+> Internationalization is the design and development of a product, application or document content that enables easy localization for target audiences that vary in culture, region, or language.
+>
+> -- <cite>[W3C](https://www.w3.org/International/questions/qa-i18n.en#i18n)</cite>
+
+<!-- ? not sure if using a citation is a good idea to introduce/define i18n — feedback is welcome! -->
+
+The Internationalization (i18n) plugin allows Strapi users to create, manage and distribute content in several languages.
+
+
+The i18n plugin:
+
+- allows admin panel users to create several localized versions of their content (see [user guide](/user-docs/latest/content-manager/translating-content.md))
+- allows developers to build localized projects by fetching and consuming the right content depending on the country/language of the audience.
+
+::: tip NOTE
+The i18n plugin does not automatically translate the users' content nor adapt the admin interface to languages specificities (like displaying the admin panel in Right To Left (RTL) format).
+:::
+
+## Installation
+
+The i18n plugin requires a Strapi application running on version 3.6.0 or higher.
```

Comment:
I thought doing so, but seems like the only prerequisite is version 3.6. 
I agree for the sake of consistency — but is it worth it having a dedicated section with just one prerequisite? cc @meganelacheny

Reply:
Well I think it's time to officially decide how we should display this kind of information, because clearly we may have other docs with other prerequisites in the future, and right now you only have 2 docs to update so it's quite easy. Maybe having a "Prerequisites" section isn't the best for the dev docs because we already have lots of titles and long docs. Perhaps you could create a new type of note with "Prerequisites" as title? What do you think @pwizla ?

Reply:
I agree @meganelacheny . By "note" do you mean a custom container (that we sometimes call "callout") [like this](https://vuepress.vuejs.org/guide/markdown.html#default-title)?

Reply:
Suggestion in [this commit](https://github.com/strapi/documentation/pull/224/commits/580ee2e9cc5bd85a8bc90488326865cb4c0d6731): 
![Screenshot 2021-04-14 at 19 12 56](https://user-images.githubusercontent.com/4233866/114751474-7a071e80-9d55-11eb-8097-c2f11d95e3b1.png)